earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -148,12 +152,9 @@ install_or_upgrade_asset_root()
         chown -R "$GEFUSIONUSER:$GEGROUP" "$ASSET_ROOT"
     else
         # upgrade asset root -- if this is a master
-        if [ "$IS_SLAVE" = false ]; then
-            # TODO: Verify this logic -- this is what is defined in the
-            # installer documentation, but needs confirmation
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gegroup_existed NEW_GEFUSIONUSER
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gefusionuser_existed NEW_GEFUSIONUSER
-            if [ "$NEW_GEGROUP" = true ] || [ "$NEW_GEFUSIONUSER" = true ]; then
+        if [ "$IS_SLAVE" = "false" ]; then
+            OWNERSHIP=`find /gevol/assets -maxdepth 0 -printf "%g:%u"`
```

Comment:
@tst-ppenev so I came up with a new and more direct way to test asset file ownership. If ownership is same as rpm expects, it then sets nochown...

Reply:
Yeah, this is what I was suggesting earlier.  I have two comments:

* Use `"$ASSET_ROOT"` instead of hard coding `/gevol/assets`.
* I'm wondering if `-maxdepth 0` is going to be enough.  You could use something like `find -maxdepth 2 \( \! -user "$GEFUSIONUSER" \) -o \( \! -group "$GEGROUP" \)` to test whether you should `chown`.